src/AvatarStack/AvatarStack.tsx

```diff
@@ -127,21 +163,42 @@ const transformChildren = (children: React.ReactNode) => {
 
 export type AvatarStackProps = {
   alignRight?: boolean
+  disableExpand?: boolean
```

Comment:
What do you think about `expand: boolean` where default is set to true?

Reply:
It feels strange to have to explicitly pass `propName={false}` because boolean props are normally evaluated as `false` if the prop is excluded.

If we didn't set `expand` to `true` by default, then `<AvatarStack />` would be the same as `<AvatarStack expand={false} />`

Reply:
> It feels strange to have to explicitly pass propName={false}

true true!

> because boolean props are normally evaluated as false if the prop is excluded

not sure about that tho 😅 

```jsx
<AvatarStack/> 
<AvatarStack expand />
<AvatarStack expand={true} />

<AvatarStack expand={false}/>
```

buuuut, happy to go with what you feel strongly about